create_infotexts.py

```diff
@@ -154,9 +154,11 @@ def generate_infobox_template(item, img, places_mapping):
     infobox += "| depicted place     = " + img.data["depicted_place"] + "\n"
 
     infobox += "| date               = "
-    if not item["Fotodatum"] == "":
-        infobox += str(item["Fotodatum"])
-    infobox += "\n"
+    if item["Fotodatum"]:
```

Comment:
Or do we maybe want to add between... even if fotodatum is None? In which case we owuld have to add the first test would become `if not item["Fotodatum"] or item["Fotodatum"] == "1927-1931"`